scripts/apply_patches.sh

```diff
@@ -0,0 +1,33 @@
+#!/bin/bash
+
+# these probably won't change
+USER="psy"
+# these will probably change over time for versioning
+#TARGETBRANCH="evt-patched-v1.1"
+TARGETBRANCH="my-local-branch"
+
+# this all assumes patches/${REPO}.patch exists
+# ...for every submodule repo
+
+# assumes we're running from top-level of EVT repo
+# go into submodules directory
+cd submodules
+for REPO in $(ls); do
+	echo "Applying patch to ${REPO}..."
+	# go into repo
+	cd ${REPO}
+	# get rid of prior patch
+	# if you've made other changes, too bad, why are you running this script
+	git reset --hard HEAD
+	# update master branch, assuming it's called "master"
+	# (a safe assumption for now, but an assumption indeed)
+	git checkout master
+	git pull
```

Comment:
We should get rid of the pull and reset to the commit we use after switching to master, so that we are always checking out the commit we have submoduled against